src/app/admin/admin-orders/admin-orders.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-admin-orders',
+  templateUrl: './admin-orders.component.html',
+  styleUrls: ['./admin-orders.component.scss']
+})
+export class AdminOrdersComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```

Comment:
this is not used so we should remove it and not have the component implement OnInit

Reply:
Do you mean that I will not use the entire component at all for the whole project?

Reply:
just the ngOnInit function. When you implement OnInit for a component you need to  add the ngOnInit lifecycle function but here you have no logic in that function so you can rip it out for simplicity.

Reply:
OK got it.  Thanks for the explanation.